state_machine_enum.gemspec

```diff
@@ -33,5 +33,7 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{\Aexe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activesupport", ">= 6.0"
+  spec.add_dependency "activesupport", "~> 7"
+  spec.add_development_dependency "sqlite3", "~> 1.4"
+  spec.add_development_dependency "rails", "~> 7"
```

Comment:
Let's not do this - depending just on activerecord is sufficient. The Gemfile.lock becomes huge and we don't need the entirety of Rails at all